Jenny's Dessert Book.html

```diff
@@ -3,37 +3,37 @@
 <html><head>
     <meta charset="-8">
         <title>Jenny's Dessert Book</title>
-        <link rel="stylesheet" href="./Jenny&#39;s Dessert Book_files/index.css">
+        <link rel="stylesheet" href="/Users/jenny/Downloads/bootcamp-project-2021/index.css">
```

Comment:
Only somebody with access to your computer will be able to access a file like this. To fix this, we can use relative file paths. So change it to href="./index.css" or href="index.css", to fix this!
Similarly, fix any other links on your site that do this!

Here is some more info about relative file paths [https://www.w3schools.com/html/html_filepaths.asp](url)